samples/StatusCodePagesSample/hosting.json

```diff
@@ -1,4 +0,0 @@
-{
-  "server": "Microsoft.AspNetCore.Server.Kestrel",
-  "server.urls": "http://localhost:5001/base"
```

Comment:
I tested the samples before merging and there was no particular reason for that specific port.